testing/test_integration

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
```

Comment:
test_integration also not right I think since they're all integration tests. What's the intended scope of what this file is supposed to do? If this is specific to orchestrated-testing then it could just be test_fot at this level.

Reply:
Oh. I intend for it to trigger all integration tests written in python using pyunit. similar to testing/test_unit